config/packages/security.yaml

```diff
@@ -58,6 +58,7 @@ security:
         # This makes the logout route accessible during two-factor authentication. Allows the user to
         # cancel two-factor authentication, if they need to.
         - { path: ^/logout, role: PUBLIC_ACCESS }
+        - { path: ^/custom-style, role: PUBLIC_ACCESS }
```

Comment:
can you put a comment here why this is needs to be publicly accessible?

Reply:
alright

Reply:
added